performance-optimisation.php

```
@@ -43,6 +43,31 @@
+ * @since 1.7.0
+ * @return void
+ */
+function wppo_maybe_upgrade(): void {
+	$stored_version = get_option( 'wppo_db_version', '0' );
+
+	if ( version_compare( $stored_version, WPPO_VERSION, '>=' ) ) {
+		return;
+	}
+
+	require_once WPPO_PLUGIN_PATH . 'includes/class-telemetry-table.php';
+	\PerformanceOptimise\Inc\Telemetry_Table::create_table();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PerformanceOptimise\Inc\Telemetry_Table' in method 'wppo_maybe_upgrade'. (undefined)

(StaticAccess)